[2025-10-01T00:00:22.160+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-01T00:00:22.166+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hw1.collect_metrics_model scheduled__2025-09-30T00:00:00+00:00 [queued]>
[2025-10-01T00:00:22.169+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hw1.collect_metrics_model scheduled__2025-09-30T00:00:00+00:00 [queued]>
[2025-10-01T00:00:22.169+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-01T00:00:22.173+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): collect_metrics_model> on 2025-09-30 00:00:00+00:00
[2025-10-01T00:00:22.176+0000] {standard_task_runner.py:64} INFO - Started process 30172 to run task
[2025-10-01T00:00:22.178+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'hw1', 'collect_metrics_model', 'scheduled__2025-09-30T00:00:00+00:00', '--job-id', '258', '--raw', '--subdir', 'DAGS_FOLDER/kudryakov_hw1.py', '--cfg-path', '/tmp/tmpy0ep4a28']
[2025-10-01T00:00:22.179+0000] {standard_task_runner.py:91} INFO - Job 258: Subtask collect_metrics_model
[2025-10-01T00:00:22.197+0000] {task_command.py:426} INFO - Running <TaskInstance: hw1.collect_metrics_model scheduled__2025-09-30T00:00:00+00:00 [running]> on host 2b301b74205f
[2025-10-01T00:00:22.228+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='VLADISLAV KUDRYAKOV' AIRFLOW_CTX_DAG_ID='hw1' AIRFLOW_CTX_TASK_ID='collect_metrics_model' AIRFLOW_CTX_EXECUTION_DATE='2025-09-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-30T00:00:00+00:00'
[2025-10-01T00:00:22.229+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-01T00:00:22.234+0000] {logging_mixin.py:188} INFO - collect_metrics_model
[2025-10-01T00:00:22.234+0000] {kudryakov_hw1.py:209} INFO - Начали расчет метрик
[2025-10-01T00:00:22.241+0000] {base.py:84} INFO - Using connection ID 's3_connection' for task execution.
[2025-10-01T00:00:22.241+0000] {connection_wrapper.py:388} INFO - AWS Connection (conn_id='s3_connection', conn_type='aws') credentials retrieved from login and password.
[2025-10-01T00:00:28.320+0000] {logging_mixin.py:188} INFO - {'r_squared': 0.8503834409713716, 'RMSE': 0.19244684232083303} metrics
[2025-10-01T00:00:28.611+0000] {kudryakov_hw1.py:228} INFO - Успешно Завершен расчет метрик
[2025-10-01T00:00:28.632+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-01T00:00:28.633+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-01T00:00:28.640+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=hw1, task_id=collect_metrics_model, run_id=scheduled__2025-09-30T00:00:00+00:00, execution_date=20250930T000000, start_date=20251001T000022, end_date=20251001T000028
[2025-10-01T00:00:28.666+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-01T00:00:28.676+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-01T00:00:28.677+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
